server/src/templates/agent_detail.html

```diff
@@ -36,6 +36,31 @@ <h2 class="p-muted-heading">Queues</h2>
     {% endfor %}
 </ul>
 
+<h2 class="p-muted-heading">Provision History</h2>
+<div>
+    <table>
+        <thead>
+            <tr>
+                <th style="width: 150pt">Timestamp</th>
```

Comment:
Is the timestamp the point at which the job ended? I think it might be good to make that clear by changing the heading to "Finished at" for example